### PR TITLE
update ruby test versions

### PR DESCRIPTION
**Pull request by kbrock**
This was a little wonky for me

So I updated the ruby versions under test and narrowed down the bundler version a little

### CODE REVIEW

**Comment by djberg96** [comment]
👍

**Comment by djberg96** [comment]
@miq-bot add_reviewer @Fryguy

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
...
 sudo: false
 language: ruby
 rvm:
-  - 2.3.1
-before_install: gem install bundler -v 1.12.5
+- 2.5.7
+- 2.6.5
+cache:
+  bundler: true
+before_install: gem install bundler -v 2.1.3
```
Comment (djberg96) [inline .travis.yml:9]: Is this line even necessary? Do we do this elsewhere and I just didn't notice?

Edit: NM, it's necessary.

Comment (kbrock) [reply .travis.yml:9]: Yea, I tried that [here](https://travis-ci.org/ManageIQ/query_relation/jobs/632311667) but it failed the build

**Review by djberg96 — APPROVED** [review]

**Pull request merged**